src/helpers/debounce.test.ts

```diff
@@ -0,0 +1,155 @@
+import GovukButton from "govuk-frontend/govuk/components/button/button";
+import React from "react";
+
+import { debounce, makeDebounceContext } from "./debounce";
+
+it("returns `false` and doesn't prevent the default action on both of two call within the default timeout without `data-prevent-double-click` set on the event's target, similar to `GovukButton.prototype.debounce`", () => {
```

Comment:
"returns `false` and doesn't prevent the default action on both of **the** two call within the default timeout when `data-prevent-double-click` isn't set on the event's target, similar to `GovukButton.prototype.debounce`"